executor/linux/step_test.go

```diff
@@ -338,9 +340,6 @@ func TestLinux_StreamStep(t *testing.T) {
 	}
 
 	_runtime, err := docker.NewMock()
-	if err != nil {
-		t.Errorf("unable to create runtime engine: %v", err)
-	}
```

Comment:
Could you share why these lines were removed?

Perhaps it was an accident? 😅

Reply:
Yes it was 😅